### ISSUE TITLE
Performance of frrnA on Cityscapes

### ISSUE THREAD

**Issue by jthsieh** [meetshah1995/pytorch-semseg#57]
Thank you for providing the code! I ran the frrnA model on Cityscapes and was hoping to replicate the results reported in the paper (FRRN A: 0.630 mean IoU),
`python train.py            
  --arch frrnA             
  --dataset cityscapes     
  --img_rows 256           
  --img_cols 512           
  --n_epoch 35             
  --batch_size 3           
  --l_rate 0.001`
And then I continue training with learning rate 1e-4 for 10 epochs.

This is roughly the same training setup and hyperparameters as in the paper. However, I only got 0.50 mean IoU.
I'm wondering if I'm doing something wrong. Were you able to replicate the results?

**Comment by meetshah1995** [comment]
Nope, I wasn't able to match the paper results. 

After careful perusal of the paper, the only difference seems be that of the optimizer (Adam [We use SGD with momentum]) and that the paper uses nearest neighbour interpolation in the FRRUs, we use bilinear interpolation.

**Comment by victorjdcom** [comment]
i am using this config file for the cityscapes:

model:
    arch: fcn8s
data:
    dataset: cityscapes
    train_split: train
    val_split: val
    img_rows: '256'
    img_cols: '512'
    path: /home/victor/data/cityscapes
training:
    train_iters: 300000
    batch_size: 1
    val_interval: 1000
    n_workers: 16
    print_interval: 50
    optimizer:
        name: 'sgd'
        lr: 1.0e-10
        weight_decay: 0.0005
        momentum: 0.99
    loss:
        name: 'cross_entropy'
        size_average: False
    lr_schedule:
    resume: fcn8s_cityscape_best_model.pkl

got error for:  "No files for split=[%s] found in %s" % (split, self.images_base)

any idea?
thanks
